tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -579,9 +570,9 @@ def test_parse_ssp_section1_datafile(ssp_section1_datafile, dfs):
     ssp_section1_datafile.year = 2019
     ssp_section1_datafile.quarter = 'Q1'
 
-    expected_m1_record_count = 820
-    expected_m2_record_count = 992
-    expected_m3_record_count = 1757
+    expected_m1_record_count = 818
```

Comment:
does this mean we have duplicate lines in this file?

Reply:
never mind, I see we have two duplicates 👍

Reply:
It either implies there are duplicate records are standard cat4 errors. Either or will force the entire case to be discarded.